cmd/start.go

```diff
@@ -48,12 +50,17 @@ var (
 		kubeconfig string
 		nodeName   string
 
+		listenAddr string
+
 		enableAutoUpdate bool
 	}
 )
 
 func init() {
+	startOpts.listenAddr = "0.0.0.0:11345"
+
 	rootCmd.AddCommand(startCmd)
+	startCmd.PersistentFlags().StringVar(&startOpts.listenAddr, "listen", startOpts.listenAddr, "Address to listen on for metrics")
```

Comment:
any reason why not inline?

Reply:
inline how?  localhost you mean?

Reply:
```golang
startCmd.PersistentFlags().StringVar(&startOpts.listenAddr, "listen", "0.0.0.0:12345", "Address to listen on for metrics")
```

Reply:
@abhinavdahiya when using a cobra approach with options structs and flags for customization (like kube-apiserver, generic-apiserver, oc, kubectl, etc), we've found that respecting a struct value allows for flexibility and clean composition.  This matches the upstream projects better.